src/Plugins/NamedPipeService/Configuration/NamedPipeServerTransportOptions.cs

```diff
@@ -0,0 +1,26 @@
+// Copyright (C) 2015-2024 The Neo Project.
+//
+// NamedPipeServerTransportOptions.cs file belongs to the neo project and is free
+// software distributed under the MIT software license, see the
+// accompanying file LICENSE in the main directory of the
+// repository or http://www.opensource.org/licenses/mit-license.php
+// for more details.
+//
+// Redistribution and use in source and binary forms with or without
+// modifications are permitted.
+
+using Neo.Plugins.Buffers;
+using System;
+using System.Buffers;
+
+namespace Neo.Plugins.Configuration
+{
+    internal class NamedPipeServerTransportOptions
+    {
+        public int ListenerQueueCount { get; set; } = Math.Min(Environment.ProcessorCount, 16);
```

Comment:
This should be configurable or less, seems that we prioritize this instead of other threads

Reply:
Sure I can make configurable. But shouldn't be. Reason being they could configure incorrectly. But I'll see what I can do.